src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -179,6 +186,14 @@ def load_command_table(self, _):
         g.custom_command('delete', 'aks_snapshot_delete',
                          supports_no_wait=True)
 
-    # AKS trusted access roles commands
+    # AKS trusted access role commands
     with self.command_group('aks trustedaccess role', trustedaccess_role_sdk, client_factory=cf_trustedaccess_role) as g:
         g.custom_command('list', 'aks_trustedaccess_role_list')
+
+    # AKS trusted access rolebinding commands
+    with self.command_group('aks trustedaccess rolebinding', trustedaccess_role_binding_sdk, client_factory=cf_trustedaccess_role_binding) as g:
+        g.custom_command('list', 'aks_trustedaccess_role_binding_list')
+        g.custom_command('get', 'aks_trustedaccess_role_binding_get')
+        g.custom_command('create', 'aks_trustedaccess_role_binding_create_or_update')
+        g.custom_command('update', 'aks_trustedaccess_role_binding_create_or_update')
+        g.custom_command('delete', 'aks_trustedaccess_role_binding_delete')
```

Comment:
Agreed with Xing. If there is nothing special to deal with, maybe use the delete command provided by the CLI and remove your custom implementation in `custom.py` instead?
```suggestion
        g.command('delete', 'delete', confirmation=True)
```

Reply:
Discussed offline, it cannot use g.command if we have custom argument like `role-binding-name`, so keep using `custom_command`